torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -312,6 +316,8 @@ std::shared_ptr<FutureMessage> TensorPipeAgent::send(
   requestMessage.setId(nextMessageID_++);
   pendingResponseMessage[requestMessage.id()] = futureResponseMessage;
 
+  ++clientActiveCalls_;
+
   // Don't need to hold lock while calling tensorpipe API.
   lock.unlock();
 
```

Comment:
GitHub won't let me comment on lines you didn't touch, but further down there are two error-handling check in which I think we should also decrease `clientActiveCalls_`. Basically, every time we do `futureResponseMessage->setError()`.

Reply:
I think there's still one occurrence you missed, on line 348.